src/openapi/streaming/streaming.js

```diff
@@ -933,6 +934,36 @@ Streaming.prototype.disconnect = function() {
     this.connection.stop();
 };
 
+Streaming.prototype.pause = function() {
+    this.paused = true;
+
+    if (this.reconnecting) {
+        clearTimeout(this.reconnectTimer);
+        this.reconnecting = false;
+        this.retryCount = 0;
+    }
+
+    this.orphanFinder.stop();
+
+    for (let i = 0; i < this.subscriptions.length; i++) {
+        const subscription = this.subscriptions[i];
+        // Mark the subscription as not having a connection and reset it so its state becomes unsubscribed
+        subscription.onConnectionUnavailable();
+        subscription.reset();

```

Comment:
is there a difference between resetting here vs resetting on resume?

Reply:
I don't think there is much difference but its cleaner in server perspective  i.e. cleanup and close ?
This will just push next action i.e. subscribe in the queue until the connection is available

Reply:
see comment above now

Reply:
ahh right